packages/config/src/layer2s/aztecconnect.ts

```diff
@@ -180,40 +134,28 @@ export const aztecconnect: Layer2 = {
         },
       ],
     },
-    proposerFailure: {
-      ...RISK_VIEW.PROPOSER_CANNOT_WITHDRAW,
-      description: `Only the whitelisted proposers can publish L2 state roots on L1 within ${escapeHatchDelayString} from the last posted root, so in the event of failure the withdrawals are frozen.`,
-      sources: [
-        {
-          contract: 'RollupProcessorV2',
-          references: [
-            'https://etherscan.io/address/0x7d657Ddcf7e2A5fD118dC8A6dDc3dC308AdC2728#code#F1#L697',
-          ],
-        },
-      ],
-    },
     validatedBy: RISK_VIEW.VALIDATED_BY_ETHEREUM,
     destinationToken: RISK_VIEW.NATIVE_AND_CANONICAL(),
-  }),
+  },
   stage: getStage(
     {
       stage0: {
         callsItselfRollup: true,
-        stateRootsPostedToL1: true,
-        dataAvailabilityOnL1: true,
+        stateRootsPostedToL1: false,
+        dataAvailabilityOnL1: false,
```

Comment:
why?

Reply:
before: the central operator posted data and state on L1
now: you have to post data on L1 yourself (but all data to advance state is available)
you can basically only advance because it is only one user at a time with their own data that processes the rollup

so i keep DA as true?

Reply:
both of them are true